index.html

```diff
@@ -27,6 +27,157 @@ <h2>I’m a software developer</h2>
         <li><a href="#"><img src="angellist.png" alt="angellist"></a></li>    
     </ul>  
     </section>
+
+    <header class="title">
+        <div><a href="title"><img src="Disabled.png" alt="scroll down button"></a></div>
+        <div><h1>My recent Work</h1></div>
+        <div><img src="indicator.png" alt="indicator"></div>
+    </header>
+    <section class="descript">
```

Comment:
- [ ] As the previous reviewer mentioned your design should be similar to the Figma design, please tweak the design so it's as close as possible to the Figma design

|Your design|Figma design|
|-|-|
|![Screenshot_20220531_054042](https://user-images.githubusercontent.com/25227022/171094189-0ccc7edd-57b6-45fb-9a19-bbcedb49c710.png)|![](https://user-images.githubusercontent.com/48371156/170841109-e104deb5-db69-4b9e-87c1-2e12e54a520e.png)|